web/packages/teleport/src/Roles/RoleEditor/StandardEditor.tsx

```diff
@@ -896,6 +927,84 @@ export function WindowsDesktopAccessSpecSection({
   );
 }
 
+export function AdminRules({
```

Comment:
I'm wondering, shouldn't we run validation before switching to the yaml view?
In the case below, our validation would highlight the missing field:
<img width="740" alt="image" src="https://github.com/user-attachments/assets/20fa2fc5-7776-4129-b9e0-b4c377bb1436">

Reply:
Ugh! You're right.